config/initializers/custom_exception_mappings.rb

```diff
@@ -17,7 +17,7 @@
   "Catalog::RBACError"                         => :service_unavailable,
   "Catalog::ServiceOfferingArchived"           => :bad_request,
   "Catalog::SourcesError"                      => :service_unavailable,
-  "Catalog::TopologyError"                     => :service_unavailable,
+  "Catalog::InventoryError"                    => :service_unavailable,
```

Comment:
@hsong-rh Should this CatalogInventory Error other we will confuse with the Inventory object error

Reply:
Changed